common/src/test/kotlin/streams/configuration/StreamsConfigProceduresIT.kt

```diff
@@ -0,0 +1,41 @@
+package streams.configuration
+
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.neo4j.kernel.impl.proc.Procedures
+import org.neo4j.kernel.internal.GraphDatabaseAPI
+import org.neo4j.test.TestGraphDatabaseFactory
+import kotlin.streams.toList
+import kotlin.test.assertEquals
+
+@Suppress("DEPRECATION")
+class StreamsConfigProceduresIT {
+
+    private lateinit var db: GraphDatabaseAPI
+
+    @Before
+    fun setUp() {
+        db = TestGraphDatabaseFactory()
+                .newImpermanentDatabaseBuilder()
+                .newGraphDatabase() as GraphDatabaseAPI
+    }
+
+    @After
+    fun tearDown() {
+        db.shutdown()
+    }
+
+    @Test
+    fun `should set properties`() {
+        db.dependencyResolver.resolveDependency(Procedures::class.java)
+                .registerProcedure(StreamsConfigProcedures::class.java, true)
+        val props = mapOf("streams.procedures.enabled" to "true")
```

Comment:
is it possible to use the dynamic config procedure to set procedures.enabled=false, and thereby break yourself and be unable to change it because the procedures are no longer available?   This would be bad.

Reply:
no this cannot happen because `streams.procedures.enabled` does not affect `streams.configuration.get/set`